lib/matplotlib/backend_tools.pyi

```diff
@@ -75,8 +75,8 @@ class ToolXScale(AxisScaleBase):
     def set_scale(self, ax, scale: str | ScaleBase) -> None: ...
 
 class ToolViewsPositions(ToolBase):
-    views: dict[Figure | Axes, cbook.Stack]
-    positions: dict[Figure | Axes, cbook.Stack]
+    views: dict[Figure | Axes, cbook._Stack]
+    positions: dict[Figure | Axes, cbook._Stack]
```

Comment:
So this is a bit weird to have here in the public API when the `cbook.Stack` is going away. mypy doesn't like this since it's not in `cbook.pyi`. Is this really something that should be public?

Reply:
We discussed on the call today and decided to keep `cbook._Stack` in the type stubs